pkg/reconciler/revision/resources/queue.go

```diff
@@ -364,6 +357,19 @@ func makeQueueContainer(rev *v1.Revision, cfg *config.Config) (*corev1.Container
 		}},
 	}
 
+	//status.podIP isn't available for downward api from AKS (and other?) virtual nodes:
+	// https://docs.microsoft.com/en-us/answers/questions/352186/pods-stuck-in-pending-state-on-virtual-nodes.html
+	if(cfg.Features.InjectPodIP == apicfg.Enabled) {
```

Comment:
a lot of the tests directly create a FeaturesConfig struct rather than creating it with `defaultFeaturesConfig` (e.g. https://github.com/knative/serving/blob/b4b3ad9ecf96d3a6d79a5bfb71ea1de4384822b6/pkg/reconciler/revision/resources/queue_test.go#L346), you would have to manually add `InjectPodIP: apicfg.Enabled` to these, or else you could (and I'd personally suggest this is better anyway) invert the flag so it's something like "DisablePodIP" so that the zero value corresponds to the current - and more common - behaviour.

Reply:
Understood.  I'll wait for your response about the more generic flag name and however you decide there I'll include flipping the behavior like you suggest here.